### DISCUSSION TITLE
Use Bridge4 as "big foot" two switch MIDI controller

### DISCUSSION THREAD

**Discussion by ultracello** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#62] [Feature Requests & Ideas]
The too small space between all switches is a problem for big feet especially when you are playing with closed eyes and want to focus in playing music instead of too much check your foot is trigger the right switch to avoid accidents hitting the wrong switches.

My goal: you can use two switches as **one** switch. In global settings you can change from current standard mode to "big foot mode". Perfect to prevent accidental kicks on the neighbor switch.

Users can use the first row of two switches (3+4) as **one** joined switch. The same for the second row of two switches (1+2).

Means: the first switch of each row has its MIDI messages, the second switch of each row is fully ignored.

**It would be super awesome if both LEDs per row would always do the same – and you can use the full amount of chars of the full line, _centered_.**

Finally I just need to build a clever anytime removable but still stable connection per two switches and Bridge4 would be the coziest but most flexible 2-switch MIDI foot controller out there with the best display. 🙏🙏🙏🙏🙏👍

**Comment by simonaglover95** [comment]
For those interested - you can use switch groups to get any BRIDGE to work like this. The switch labels do not merge or centre, but you can trigger multiple switches by pressing one switch and any switch in the group can trigger that function.
